config/datatypes_conf.xml.sample

```diff
@@ -729,6 +729,9 @@
     <sniffer type="galaxy.datatypes.binary:Biom2"/>
     <sniffer type="galaxy.datatypes.binary:H5"/>
     <sniffer type="galaxy.datatypes.binary:Bam"/>
+    <sniffer type="galaxy.datatypes.binary:BamQuerynameSorted" />
+    <sniffer type="galaxy.datatypes.binary:BamInputSorted" />
```

Comment:
Thanks for this, I missed that! Can you move `galaxy.datatypes.binary:BamInputSorted`  below `BamNative` ? We don't want to actually sniff that one (or give the impression that it can be sniffed).

Reply:
Ugh - this was my attempt to fix the failing test before you did. This commit doesn’t belong here - sorry about that and thanks for catching it.

Reply:
Okay - I've stripped the commit. Thanks again for the catch.